Makefile

```diff
@@ -74,16 +74,17 @@ EXES = \
 #   any of their names
 SCRIPTS = $(shell ls scripts | grep -v -E '([\#\~]|\.orig|\.bak|\.BAK)' )
 LIBS = lcovutil.pm
+MAN_SECTIONS = 1 5
 # similarly, lets not talk about man pages
-MANPAGES = $(foreach m, $(shell cd man ; ls *.1), man1/$(m)) \
-	$(foreach m, $(shell cd man ; ls *.5), man5/$(m))
+MANPAGES = $(foreach s, $(MAN_SECTIONS), $(foreach m, $(shell cd man ; ls *.$(s)), man$(s)/$(m)))
 
 # Program for checking coding style
 CHECKSTYLE = $(CURDIR)/bin/checkstyle.sh
 
 INSTALL = install
 FIX = $(realpath bin/fix.pl)
-RM = rm
+RM = rm -f
+RMDIR = rmdir --ignore-fail-on-non-empty
```

Comment:
`--ignore-fail-on-non-empty` is likely not POSIX (https://pubs.opengroup.org/onlinepubs/009695399/utilities/rm.html)